README.md

```diff
@@ -47,6 +47,18 @@ This will open a furniture store complete with all of the props. Select an item
 
 > Note: The place on ground button sometimes does not work properly depending on where the native detects the ground to be.
 
+## Dynamic Doors Preview & Setup
+
+https://github.com/complexza/ps-housing/assets/74205343/fef28197-8e95-4b5f-aba8-b3e5409456d8
+
+https://github.com/complexza/ps-housing/assets/74205343/196bb25e-75b9-42f6-b1c3-fe86ad477544
+
+### Setup
+- You will need to set the `Config.EnableDynanicDoors = true`
+- You will have to add this convar into your server.cfg `setr game_enableDynamicDoorCreation "true"`
```

Comment:
Curious, why is this convar needed?

Reply:
I am actually not sure, I just knows it makes the doors dynamic, as far as I know the convar just converts the doors into the dynamic mode

Reply:
> Added game_enableDynamicDoorCreation network check bypass, allowing the creation of doors dynamically in a network environment (thanks to [@Nikez](https://forum.cfx.re/u/nikez), [@z3t4s](https://forum.cfx.re/u/z3t4s), and okqut).

I found this when it was added https://forum.cfx.re/t/cfx-re-community-pulse-june-2023-edition/5109543

Reply:
ohh sick! Thanks, I struggled to find docs on it!